AlphaWallet/Tokens/Helpers/CachedERC1155ContractDictionary.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  CachedERC1155ContractDictionary.swift
+//  AlphaWallet
+//
+//  Created by Jerome Chan on 12/4/22.
+//
+
+import Foundation
+
+class CachedERC1155ContractDictionary {
+    private var baseDictionary: [AlphaWallet.Address: Bool] = [AlphaWallet.Address: Bool]()
+    private var fileUrl: URL
+    private var encoder: JSONEncoder
+
+    static func loadFromCache(fileName: String) -> CachedERC1155ContractDictionary? {
+        do {
+            var url: URL = try FileManager.default.url(for: .cachesDirectory, in: .userDomainMask, appropriateFor: nil, create: true)
```

Comment:
Should we be using `cacheUrlFor(fileName: String)` here instead?

Reply:
`cacheUrlFor(fileName: String)` is just used for testing. Do you want me to move it to the project?

Reply:
Oh, I see what you mean. Never mind that then.